datasetinsights/datasets/coco.py

```diff
@@ -1,22 +1,12 @@
 import fcntl
```

Comment:
Do we need to keep coco.py file here? Are we keeping the dataset class in this repo?

Reply:
As discussed earlier, we are keeping only those dataset subclasses which do not have dependencies on tf and torch. 
- coco is not a subclass of Dataset, its just preprocess the downloaded data.

Reply:
So, what part of the datasetinsights package do we want users to use? I see inconsistent patterns here for Coco and GroceriesReal dataset. One is still a dataset class the other only have preprocess method and capability to load coco dataset from disk. 

I suggest we think about the design of dataset class (and dataset catalog object) more carafully and decide what module we keep and what we remove.